### ISSUE TITLE
Contrast - contact

### ISSUE THREAD

**Issue by vavroom** [nicolas-steenhout/main#110]
◦	Error message contrast is insufficient in light mode and dark mode
◦	Links for the email address and phone number have insufficient contrast in light mode
